content/5_Plat/Binary_Jump.mdx

```diff
@@ -142,14 +142,14 @@ int main() {
 
 <Resources>
 	<Resource source="CPH" title="18.3 - LCA Method 1">
-		brief description
+		Brief description/solution.
```

Comment:
```suggestion
		Brief description/solution
```
It's a fragment so I don't think a period is suitable? & resources on other sections don't have periods.